### ISSUE TITLE
Default tab title too long

### ISSUE THREAD

**Issue by nurbles** [microsoft/terminal#2327]
The default title for my powershell tabs is so long that (a) it is cut off and (b) the 'X' to close the tab is not displayed, for example:

![wt-title](https://user-images.githubusercontent.com/14079927/62626759-14af2900-b8f6-11e9-83df-85906d423665.png)

# Environment

```none
Windows build number: Microsoft Windows [Version 10.0.18362.239]
Windows Terminal version (if applicable):  0.3.2171.0

Any other software? no
```
I did edit the settings to make two changes:  The point size for all fonts is now 12 instead of 10 and CMD is the default instead of powershell.

# Steps to reproduce

Open a powershell tab in Windows Terminal

# Expected behavior

At a minimum, I would expect the tab close control to remain visible or at least a click where the control *belongs* should close the tab.

# Actual behavior

The long path appears to overwrite the close tab control and prevent any clicks from reaching the close tab control.

**Comment by mdtauk** [comment]
The TabView control template for the tab items, should ensure the close button gets priority as the control scales down.

**Comment by DHowett-MSFT** [comment]
@mdtauk that is an excellent thing to file over on microsoft/microsoft-ui-xaml./

**Comment by DHowett-MSFT** [comment]
In the mean time, override the title with `tabTitle` or change your shell configuration to set a better title.

**Comment by DHowett-MSFT** [comment]
#608

**Comment by nurbles** [comment]
I don't really see the duplication as I was reporting that the close tab control was being overwritten by a long title, while #608 seems to be about configuring custom titles.  However, I can understand blaming something else, I do that all the time myself.  :-)

FWIW, If I add and remove more tabs, the tab with an over-long title will get 'fixed' to show the whole title and the close control.  I have been able to get a new tab to obscure the close control with much shorter tab titles, too, by adding tabs of different types -- it rarely takes more than one of two attempts, no matter which shell type I choose.  Oh well.  I tried.
